release/resources/module_list.json

```diff
@@ -37,6 +37,9 @@
             "name": "module-ballerinax-java.jdbc",
             "version_key": "stdlibJdbcVersion"
         },
+        {
+            "name": "module-ballerinax-java.jms"
```

Comment:
We need to add a version key for this. [Reference](https://github.com/ballerina-platform/ballerina-standard-library/blob/main/docs/adding-a-new-ballerina-module.md#the-version_key-field)

Reply:
Fixed with [247c70d255cfb30d8f6011eafd31cddc63f41741](https://github.com/ballerina-platform/ballerina-standard-library/pull/4729/commits/247c70d255cfb30d8f6011eafd31cddc63f41741)